db/migrate/20150629202535_create_reviews.rb

```diff
@@ -0,0 +1,11 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.integer :rating, null: false
+      t.text :description
+      t.integer :beach_id, null: false
```

Comment:
can a user review the same beach more than once? If not, we should add a unique index for `[:beach_id, :user_id]` and validate uniqueness